tensorflow/core/platform/cpu_feature_guard.cc

```diff
@@ -138,8 +138,16 @@ void InfoAboutUnusedCPUFeatures() {
     CheckIfFeatureUnused(CPUFeature::FMA, "FMA", missing_instructions);
 #endif  // __FMA__
 #endif  // else of if defined(_MSC_VER) && !defined(__clang__)
-    if (!missing_instructions.empty()) {
-      LOG(INFO) << "This TensorFlow binary is optimized with Intel(R) MKL-DNN "
+
+    string intel_library_official_name(
+        "Intel(R) oneAPI Deep Neural Network Library (oneDNN) ");
+#ifndef INTEL_MKL
+    intel_library_official_name = "oneAPI Deep Neural Network Library (oneDNN) ";
```

Comment:
I think you meant MKL-DNN here? (Non-MKL build is still using v0.x)
```suggestion
        "Intel(R) oneAPI Deep Neural Network Library (oneDNN) ");
#ifndef INTEL_MKL
    intel_library_official_name = "Intel(R) MKL-DNN ";
```

Reply:
The string came officially from Intel customer/Marketing. So looks like that is what they want.

Reply:
But then the only difference between the two strings is "Intel (R)" (one with, and one without). Why can't both just be "Intel(R) oneAPI Deep Neural Network Library (oneDNN) "?

Reply:
Open source oneDNN does not require Intel(R). Intel branding applies to binary distribution only. So for Tensorflow we should stick with `oneAPI Deep Neural Network Library (oneDNN)`.

Reply:
I see. Thank you for the explanation!

Reply:
thank you @penpornk @vpiragov